.circleci/config.yml

```diff
@@ -385,24 +389,40 @@ workflows:
           name: distcheck_almalinux_8
           dist: almalinux
           release: "8"
+      # fedora is our witness
       - distcheck:
           name: distcheck_fedora_latest
```

Comment:
rename to 'witness_fedore_latest'?

Reply:
Not if we rename them all to `build_<dist>_<release>`?